.travis.yml

```diff
@@ -152,24 +151,41 @@ base_acceptance: &acceptance_default
 matrix:
   fast_finish: true
   include:
-    # Lint python and javascript together
-    - python: 2.7
-      name: 'Linter'
-      env: TEST_SUITE=lint
+    - language: generic
+      name: 'Linter (Javascript)'
+      env: TEST_SUITE=lint-js
       install:
-        - python setup.py install_egg_info
-        - SENTRY_LIGHT_BUILD=1 pip install -U -e ".[dev]"
+        # XXX: Under a "generic" language environment, travis pyenv is still present.
+        # .python-version will make it error because there is no pyenv python installed.
+        - rm .python-version
```

Comment:
Hmm is this better than just making the language env = python2.7?

Reply:
I'd say so. Doesn't really make sense to have python language for a job that lints JS. Also this keeps me (at least, hopefully it's helpful to other people as well) aware of `.python-version` and its influence on `pyenv`.